src/components/ListItems/ListItems.component.tsx

```diff
@@ -0,0 +1,17 @@
+import { List } from '@mui/material';
+
+import { UserProductListItem } from '@components/UserProductListItem';
+
+import { ListItemsPropsType } from './ListItems.types';
+
+export const ListItems = ({ itemsData }: ListItemsPropsType) => (
+    <List sx={{ padding: 0 }}>
```

Comment:
```suggestion
    <List disablePadding>
```
Whenever using any component please explore about its bulit in properties instead of always using styles to override